tools/amplicon_architect.cwl

```diff
@@ -10,16 +10,14 @@ requirements:
   - class: ResourceRequirement
     ramMin: 16000
     coresMin: $(inputs.threads)
-baseCommand: [mkdir, data_repo]
+  - class: InitialWorkDirRequirement
+    listing: [$(inputs.data_repo)]
+baseCommand: [export]
```

Comment:
I'd almost rather have no baseCommand in this scenario. just toss the export into the argument.